Shaders/Letterbox.fx

```diff
@@ -0,0 +1,68 @@
+// Shader by Jacob Maximilian Fober
+// https://creativecommons.org/publicdomain/mark/1.0/
+// This work is free of known copyright restrictions.
+// Letterbox PS
```

Comment:
Doesn't Border.fx already accomplish the same?

Reply:
Yes It does, I missed that.
Letterbox deleted.